app/renderer/components/frame/frame.js

```diff
@@ -316,7 +323,8 @@ class Frame extends React.Component {
         // In this case both the user display and the user think they're on this.props.location.
         if (this.props.tabUrl !== this.props.location &&
           !this.isAboutPage() &&
-          !isTorrentViewerURL(this.props.location)) {
+          !isTorrentViewerURL(this.props.location) &&
+          !isIPFSPath(this.props.location)) {
```

Comment:
I believe I got this code path right (it checks if location starts with `dweb:` or `ipfs://`), however I never managed to get the log on `74` to print.